### PR TITLE
Ticket: Visualization 

### PR DESCRIPTION
**Pull request by gjuresic**
changed based on TJ comments - using plot_lines_by function
- one function for getting predictors out of dict
- one function for plotting the lines of predictors

### CODE REVIEW

**Review by Tejas-Choudekar — APPROVED** [review]

File: exploratory_data_analysis/eda.py [hunk 1]
```diff
@@ -20,6 +20,12 @@ def main(config: DictConfig) -> None:
     data_dict = read_data(data_dir=SDKConfig().data_dir, config=config)
     processed_data_dict = preprocess_data(data_dict=data_dict)
 
+    # Visualization of predictors
+    predictor_names = get_predictor_names(processed_data_dict=processed_data_dict)
+    sdk_config = SDKConfig()
+    path_to_results = sdk_config.get_output_dir("plots_predictors")
```
Comment (cristian-castro-a) [inline exploratory_data_analysis/eda.py]: You can use it a single line of code: 
path_to_results = SDKConfig().get_output_dir('plots_predictors')

File: utils/exploratory_data_analysis.py [hunk 2]
```diff
@@ -42,4 +45,40 @@ def get_eda_summary(processed_data_dict: Dict) -> pd.DataFrame:
         ]
         summary[df_name] = results
 
-    return pd.DataFrame.from_dict(summary, orient='index', columns=columns)
\ No newline at end of file
+    return pd.DataFrame.from_dict(summary, orient='index', columns=columns)
+
+
+def get_predictor_names(processed_data_dict: Dict):
```
Comment (cristian-castro-a) [inline utils/exploratory_data_analysis.py]: Do we need this function if we have the predictor names in the config file? What's the difference between this and:
predictor_names = config['datafiles'].keys()

Remember that the predictor names are the same as the keys in config.

File: utils/exploratory_data_analysis.py [hunk 3]
```diff
@@ -42,4 +45,40 @@ def get_eda_summary(processed_data_dict: Dict) -> pd.DataFrame:
...
+    Returns:
+        None but plots the times series
+    """
+    checked_data_dict = check_correct_structures(processed_data_dict=processed_data_dict)
+
+    # Iterate over the dictionary
+    for i, (df_name, df) in enumerate(checked_data_dict.items()):
+        file_name = df_name + ".html"
+        plot_lines_by(data=df, plot_x='date', plot_y=predictor_names[i], path_to_results=path_to_results,
+                      file_name=file_name)
```
Comment (cristian-castro-a) [inline utils/exploratory_data_analysis.py:65]: Nice function!

**Review by cristian-castro-a — CHANGES_REQUESTED** [review]

File: utils/exploratory_data_analysis.py (hunk 3 above)
Comment (gjuresic) [reply utils/exploratory_data_analysis.py:65]: Thanks!

**Review by cristian-castro-a — APPROVED** [review]

**Pull request merged**